src/crimson/os/seastore/seastore_types.cc

```diff
@@ -92,6 +92,10 @@ std::ostream& operator<<(std::ostream& out, segment_seq_printer_t seq)
   }
 }
 
+std::ostream &operator<<(std::ostream &out, const laddr_t &laddr) {
+  return out << std::hex << laddr.value << std::dec;
```

Comment:
How about:
```c++
return out << "L" << std::hex << laddr.value << std::dec;
```
?

This should be helpful to make laddr explict in the logs.

Reply:
Yeah, will update in next push together with another comment.